ocs/ocp.py

```diff
@@ -134,23 +136,42 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
         return self.exec_oc_cmd(command)
 
-    def wait_for_resource_status(
-        self, resource_name, condition, timeout=30, sleep=3
+    def watch(
```

Comment:
Isn't this functionality same as "oc wait" ?

Reply:
That's true!

My intention was to split into watch() and wait() that will use watch - fixing